server/appservices/dev.go

```diff
@@ -0,0 +1,17 @@
+package appservices
+
+import (
+	"github.com/mattermost/mattermost-plugin-apps/server/config"
+	"github.com/mattermost/mattermost-plugin-apps/server/incoming"
+	"github.com/mattermost/mattermost-server/v6/model"
+)
+
+func (a *AppServices) RefreshBindings(r *incoming.Request, userID string) error {
+	err := r.RequireSourceApp()
```

Comment:
Given that a app specific session is required the app needs to persist a token during restart. How would that be accomplished in a secure and easy way?

Reply:
@hanzei That's a good point. I figured the developer would capture it themselves and save it locally for use. There should be a more straightforward way to use this though. Do you see something that can change here to make this more accessible?

Reply:
Anything that is "dev" can be scripted as part of the apps makefile, and use a sysadmin token + `curl`; 2/5 persisting app session tokens should be strongly discouraged.

Reply:
Also, I don't understand the value of refreshing a specific user's bindings on an app restart, shouldn't all users be refreshed?

Reply:
> Also, I don't understand the value of refreshing a specific user's bindings on an app restart, shouldn't all users be refreshed?

@levb This is meant to limit the scope of how many binding requests come in when this dev route is hit. In a "production" server running with dev mode on (should we even support this case?), this route could be abused to queue up a large number of requests from all users simultaneously, and repetitively. If we don't want to be concerned with this, we can instead have the endpoint have all users refresh.

Reply:
> Do you see something that can change here to make this more accessible?

Does it make sense to have an API endpoint that a user can hit to force a bindings request for themself? Then we can hit that endpoint as part of re-running the app.

Reply:
@hanzei That sounds good, but how does the App end up sending this request? Does it need a user token for this?

We could use a personal access token approach for the development side of this. The developer would create a personal access token for the user that they're using for testing, and use the token on App startup, to trigger the binding refresh for that specific user.

Reply:
Yes, a developer would use a personal access token.

To be clear: My idea was around triggering the "refresh bindings" endpoint as part of the build process, e.g. as part of the `Makefile`. I'm hesitant to put that code into the app as it's more of a "development detail".

Reply:
@hanzei This idea makes a lot of sense. Thanks for suggesting this!